src/Ticket_Contract.php

```diff
@@ -119,6 +119,7 @@ public static function displayTabContentForItem(
                 'used'        => $used,
                 'displaywith' => ['id'],
                 'entity'      => $item->fields['entities_id'],
+                'from'        => self::class
```

Comment:
I think there should be no limit in ticket/contract linking.
```suggestion
                'nochecklimit' => true,
```

Reply:
I have absolutely no idea, as said in local channel.